docs/howto/upload.adoc

```diff
@@ -14,11 +14,21 @@ qDup has an internal state variable `ENV.SCRIPT_DIR` that refers to the parent d
 If the files are stored in the same folder as the scripts then the `upload` example becomes
 
 ```yaml
-- upload: ${{ENV.SCRIPT_DIR}}/*.json /tmp/
+- upload: "${{ENV.SCRIPT_DIR}}/*.json" /tmp/
 ```
 
-Most of our script repositories have a separate folder called `assets` for files we sent to the remote computers. For those scripts, `uploads` are slightly different:
+Most of our script repositories have a separate folder called `assets` for files we send to the remote computers. For those scripts, `upload` is slightly different:
 
 ```yaml
-- upload: ${{ENV.SCRIPT_DIR}}/../assets/*.json /tmp/
+- upload: "${{ENV.SCRIPT_DIR}}/../assets/*.json" /tmp/
 ```
+
+We can also explicitly set the local file path and the upload destination in yaml
+
+```yaml
+- upload:
+    path: "${{ENV.SCRIPT_DIR}}/../assets/*.json"
+    destination: /tmp/
+```
+
+This format might be easier to read or help if the `path` or `destionation` have spaces
```

Comment:
Wild-card description.

But, more importantly - it sounds like a reference guide. HOWOTOs are specific "Upload all .json files from test suite run to /tmp"